VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoCosmosSwitch.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  TransactionMemoCosmosIBC.swift
+//  VultisigApp
+//
+//  Created by Enrique Souza Soares on 17/05/24.
+//
+
+import SwiftUI
+import Foundation
+import Combine
+
+/**
+
+ 2) COSMOS - FUNCTION: “SWITCH THORCHAIN”
+
+ UI Elements:
+     •    Address Field:
+     •    Prefilled with the user’s THORChain address
+     •    Allow manual override
+     •    Amount Field: Enter amount to switch
+
+ Action:
+ → Send MsgSend from COSMOS to THORChain vault
+ → Include memo: SWITCH:<thorAddress>
+ 
+ */
+
+class TransactionMemoCosmosSwitch: TransactionMemoAddressable, ObservableObject {
+    @Published var amount: Double = 0.0
+    @Published var destinationAddress: String = ""
+    @Published var thorAddress: String = ""
+    
+    @Published var amountValid: Bool = false
+    @Published var destinationAddressValid: Bool = true
```

Comment:
since `destinationAddress` is empty at the beginning , this should be set to `false` 
and only set it true once we get a valid inbound address from THORChain endpoint